GNOME #112172; move VteReaper down the library stack
GNOME #320127; WONTFIX upstream, we'll need to remove this eventually,
but we need to take care that nothing in Debian uses it:
gdebi: #574168

diff -urN vte-0.20.0.orig/python/vte.defs vte-0.20.0/python/vte.defs
--- vte-0.20.0.orig/python/vte.defs	2009-03-05 15:06:07.000000000 +0100
+++ vte-0.20.0/python/vte.defs	2009-04-07 20:17:08.000000000 +0200
@@ -7,6 +7,14 @@
   (gtype-id "VTE_TYPE_TERMINAL")
 )
 
+(define-object Reaper
+  (in-module "Vte")
+  (parent "GObject")
+  (c-name "VteReaper")
+  (gtype-id "VTE_TYPE_REAPER")
+)
+
+
 ;; Enumerations and flags ...
 
 (define-enum TerminalEraseBinding
@@ -813,6 +821,17 @@
 
 
 
+;; From ../src/reaper.h
+(define-function vte_reaper_get_type
+  (c-name "vte_reaper_get_type")
+  (return-type "GtkType")
+)
+ 
+(define-function reaper_get
+  (c-name "vte_reaper_get")
+  (return-type "VteReaper*")
+)
+
 ;; From vteversion.h
 
 
diff -urN vte-0.20.0.orig/python/vtemodule.c vte-0.20.0/python/vtemodule.c
--- vte-0.20.0.orig/python/vtemodule.c	2009-03-05 15:06:07.000000000 +0100
+++ vte-0.20.0/python/vtemodule.c	2009-04-07 20:18:05.000000000 +0200
@@ -25,6 +25,7 @@
 #include <pygobject.h>
 #include <pygtk/pygtk.h>
 #include "../src/vte.h"
+#include "../src/reaper.h"
 
 extern void pyvte_register_classes(PyObject * d);
 extern void pyvte_add_constants(PyObject *module, const gchar *strip_prefix);
diff -urN vte-0.20.0.orig/python/vte.override vte-0.20.0/python/vte.override
--- vte-0.20.0.orig/python/vte.override	2009-03-05 15:06:07.000000000 +0100
+++ vte-0.20.0/python/vte.override	2009-04-07 20:17:45.000000000 +0200
@@ -7,6 +7,7 @@
 #include <pygobject.h>
 #include <gtk/gtk.h>
 #include "../src/vte.h"
+#include "../src/reaper.h"
 
 void pyvte_add_constants(PyObject *module, const gchar *strip_prefix);
 void pyvte_register_classes(PyObject *d);
@@ -16,6 +17,7 @@
 import gtk.gdk.Pixbuf as PyGdkPixbuf_Type
 import gtk.MenuShell as PyGtkMenuShell_Type
 import gtk.Widget as PyGtkWidget_Type
+import gobject.GObject as PyGObject_Type
 %%
 override vte_terminal_feed kwargs
 static PyObject *
